.github/workflows/test-OSW.yml

```diff
@@ -7,12 +7,20 @@ on:
     branches: [ master, develop ]
 
 jobs:
+  find-languages:
+    runs-on: ubuntu-latest
+    steps:
+      - id: set-matrix
+        run: echo "::set-output name=languages_matrix::$(curl https://api.github.com/repos/open-smartwatch/open-smartwatch-os/contents/include/locales | jq -c '[.[].name[0:-2]]')"
```

Comment:
You are referencing this repository using a hard-coded URI. Please change it to use some of the predefined GitHub Actions variables (https://docs.github.com/en/actions/learn-github-actions/environment-variables) and also to always refer to the files of the commit it is coming from.

I strongly advise you to use your Ubuntu shell of that job to parse the files of the repository from this GitHub Action run instead. It is local, does not need any external requests and should be more future-proof anyways.

Reply:
Maybe use this bash-command instead?
```bash
cd include/locales/; X=$(ls *.h -1 | sed -e 's/\.h$//' | tr '\n' ' '); X=($X); jq --compact-output --null-input '$ARGS.positional' --args -- "${X[@]}"
```

This would be in YAML:
```yaml
- id: set-matrix
        run: echo "::set-output name=languages_matrix::$(cd include/locales/; X=$(ls *.h -1 | sed -e 's/\.h$//' | tr '\n' ' '); X=($X); jq --compact-output --null-input '$ARGS.positional' --args -- "${X[@]}")"
```